### PR TITLE
Remove Transfer model and rename database table and columns for trans…

### PR DESCRIPTION
**Pull request by clark800**
…fers

### CODE REVIEW

File: src/models/accounts.js [hunk 1]
```diff
@@ -19,66 +20,6 @@ function getPublicData (data) {
   }
 }
 
-function convertFromExternal (data) {
```
Comment (MatthewPhinney) [inline src/models/accounts.js]: I this account stuff left over from another PR?

Comment (clark800) [inline src/models/accounts.js]: No, I moved these to make it consistent with the transfers changes.

File: src/models/transfers.js [hunk 2]
```diff
@@ -121,7 +124,7 @@ function sha512 (str) {
 }
 
 function updateTransferObject (originalTransfer, transfer) {
-  let updatedTransferData = originalTransfer.getData()
+  let updatedTransferData = _.cloneDeep(originalTransfer)
```
Comment (MatthewPhinney) [inline src/models/transfers.js]: Do you need to clone this before calling `_.omit`?

File: src/models/transfers.js [hunk 3]
```diff
@@ -376,7 +378,16 @@ function * fulfillTransfer (transferId, fulfillmentUri) {
...
-function * setTransfer (transfer, requestingUser) {
+function * setTransfer (externalTransfer, requestingUser) {
+  const validationResult = validator.create('Transfer')(externalTransfer)
+  if (validationResult.valid !== true) {
+    const message = validationResult.schema
+      ? 'Body did not match schema ' + validationResult.schema
+      : 'Body did not pass validation'
+    throw new InvalidBodyError(message, validationResult.errors)
+  }
+  let transfer = converters.convertToInternalTransfer(externalTransfer)
```
Comment (MatthewPhinney) [inline src/models/transfers.js]: Why can't you use `const` here?

File: src/models/transfers.js (hunk 2 above)
Comment (clark800) [inline src/models/transfers.js]: Yeah, because I think `omit` doesn't do a deep copy.

File: src/models/transfers.js (hunk 3 above)
Comment (clark800) [inline src/models/transfers.js]: Assigned to here: https://github.com/interledger/five-bells-ledger/pull/261/files/2606625fd12d06c70ed07ab40b6be82dfe6e258f#diff-b29546dff0ad4cfb82a7d138568c73f6R423

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**